e2e/test/scenarios/custom-column/custom-column-reproductions.cy.spec.js

```diff
@@ -2056,3 +2059,86 @@ describe("Issue 58230", () => {
     H.popover().button("Done").should("be.enabled");
   });
 });
+
+describe("issue 57674", () => {
```

Comment:
I don't think we need to enumerate possible cases with an e2e test. Either no e2e test, or just one case